src/OnboardingSPA/components/StartOptions/index.js

```diff
@@ -0,0 +1,94 @@
+import { SITEGEN_FLOW } from '../../data/flows/constants';
+import { resolveGetDataForFlow } from '../../data/flows';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { validateFlow } from '../../data/flows/utils';
+import { useNavigate } from 'react-router-dom';
+import { store as nfdOnboardingStore } from '../../store';
+import getContents from '../../steps/TheFork/contents';
```

Comment:
If we want to use the contents file let's pass that instead of referencing directly as now this component cannot be reused as it will always refer to the same `contents` file.